src/dr/app/beauti/options/PartitionClockModel.java

```diff
@@ -554,8 +554,9 @@ public List<Operator> selectOperators(List<Operator> operators) {
         List<Operator> ops = new ArrayList<Operator>();
 
         if (options.hasData()) {
-            if (getDataType().getType() != DataType.TREE) {
+            // if (getDataType().getType() != DataType.TREE) {
```

Comment:
Wary of logic being commented out - will this break something else?

Reply:
I think previously, DataType.TREE was only used for empirical trees which didn't use these operators, but now DataType.Tree is true for thorney beast as well and that uses clock models. Above this line we have 

```
        if (getPartitionTreeModel().isUsingEmpiricalTrees()) {
            // empirical trees has no clock model
            return operators;
        }
 ```
 where we check if we are using empirical trees and return so we shouldn't hit this unless we are not using empirical trees, but we still want clock models if we are using thorney beast (in which case dataType will be DataType.Tree).

But some of the clock models rely on gradients and those won't work with thorney beast because there isn't sequence data. I'm not sure if this is the right place to introduce that logic.